dev-requirements.txt

```
@@ -1,11 +1,11 @@
 black==24.10.0; python_version > "3.8"
-black==24.8.0; python_version <= "3.8"
-isort; python_version < "3.8"
-isort==5.13.2; python_version >= "3.8"
-pre-commit; python_version <= "3.8"
+black==24.8.0;
```

```suggestion
black==24.10.0;
```
You'll want to delete line 1 too -- right now you're telling it to install black 24.8.0 *and* 24.10.0 which isn't going to work.